scripts.go

```diff
@@ -100,7 +100,7 @@ fi
 # Run the git command
 %v "$@"
 
-`, shell, shell, gitBinary))
+`, shell, os.TempDir(), shell, os.TempDir(), os.TempDir(), os.TempDir(), gitBinary))
```

Comment:
since same value is repeated multiple times, how about using `{temp_dir}` as placeholder and using `strings.Replacer` ?